lib/src/neo4j-client.h.in

```diff
@@ -1220,9 +1220,8 @@ int neo4j_config_set_basic_auth_callback(neo4j_config_t *config,
  *
  * @param [config] The neo4j client configuration to update.
  * @param [path] The path to the PEM file containing the private key
- *         and certificate chain. This string should remain allocated whilst
- *         the config is allocated _or if any connections opened with the
- *         config remain active_.
+ *         and certificate chain. The string will be
+ *         duplicated, and thus may point to temporary memory.
```

Comment:
Possibly doesn’t need to be explicitly stated as other methods assume string pointers can be temporary when not otherwise explicitly stated.

Reply:
It was explicitly stated in some other functions (like username setting) so I did it here as well.

Reply:
Right you are!